frontend/lib/ifixit-api/devices.ts

```diff
@@ -7,6 +7,9 @@ export async function fetchDeviceWiki(
 ): Promise<DeviceWiki | null> {
    const deviceHandle = getDeviceHandle(deviceTitle);
    try {
+      if(!deviceHandle) {
+         throw new Error(`deviceHandle cannot be a blank string!`);
+      }
```

Comment:
```suggestion
     invariant(deviceHandle.length > 0, "deviceHandle cannot be a blank string");
```

Reply:
Have you seen cases where this is called with a blank string @danielcliu-ifixit?